src/lib/stores/marketplace.ts

```diff
@@ -0,0 +1,686 @@
+        vcsProvider: 'github',
+        providerRepositoryId: 'templates',
+        providerOwner: 'appwrite',
+        providerBranch: 'main',
+        variables: [
+            {
+                name: 'VONAGE_API_KEY',
+                description: `API Key to use the Vonage API. <a class="u-bold" target="_blank" href="https://api.support.vonage.com/hc/en-us/articles/204014493-How-do-I-find-my-Voice-API-key-and-API-secret-">Learn More</a>`,
+                value: '',
+                placeholder: '62...97',
+                required: true
+            },
+            {
+                name: 'VONAGE_API_SECRET',
+                description: `Secret to use the Vonage API. <a class="u-bold" target="_blank" href="https://api.support.vonage.com/hc/en-us/articles/204014493-How-do-I-find-my-Voice-API-key-and-API-secret-">Learn More</a>`,
+                placeholder: 'Zjc...5PH',
+                required: true
+            },
+            {
+                name: 'VONAGE_API_SIGNATURE_SECRET',
+                description: `Secret to verify the JWT token sent by Vonage. <a class="u-bold" target="_blank" href="https://developer.vonage.com/en/getting-started/concepts/signing-messages">Learn More</a>`,
+                placeholder: 'NXOi3...IBHDa',
+                required: true
+            },
+            {
+                name: 'VONAGE_WHATSAPP_NUMBER',
+                description: `Vonage WhatsApp number to send messages from. <a class="u-bold" target="_blank" href="https://api.support.vonage.com/hc/en-us/articles/4431993282580-Where-do-I-find-my-WhatsApp-Number-Certificate-">Learn More</a>`,
+                placeholder: '+14000000102',
+                required: true
+            }
+        ]
+    },
+    {
+        icon: 'icon-bell',
+        id: 'push-notification-with-fcm',
+        name: 'Push Notification with FCM',
+        tagline: 'Send push notifications to your users using Firebase Cloud Messaging (FCM).',
+        permissions: ['any'],
+        events: [],
+        cron: '',
+        timeout: 15,
+        runtimes: [
+            {
+                name: 'node-18.0',
+                commands: 'npm install',
+                entrypoint: 'src/main.js',
+                providerRootDirectory: 'node/push-notification-with-fcm'
+            }
+        ],
+        instructions: `For documentation and instructions check out [README.md](https://github.com/appwrite/templates/tree/main/node/push-notification-with-fcm) file.`,
+        vcsProvider: 'github',
+        providerRepositoryId: 'templates',
+        providerOwner: 'appwrite',
+        providerBranch: 'main',
+        variables: [
+            {
+                name: 'FCM_PROJECT_ID',
+                description: `A unique identifier for your FCM project. <a class="u-bold" target="_blank" href="https://firebase.google.com/docs/projects/learn-more#project-id">Learn More</a>`,
```

Comment:
Please fix the `Learn more`, needs to be sentence case.

Also missing periods in every description after `Learn more</a>.